src/xenia/app/emulator_window.cc

```diff
@@ -118,6 +118,7 @@ bool EmulatorWindow::Initialize() {
       case ui::VirtualKey::kF5: {
         GpuClearCaches();
       } break;
+#if 0
```

Comment:
I'd say make these `#ifdef DEBUG` instead, since end-users are using the release build.